templates/components/news-list/news-item-list.twig

```diff
@@ -68,7 +68,7 @@
     {# Add the image and link only if they both exist. #}
     {% if news_list_image|render|striptags('<drupal-render-placeholder><img><picture>')|trim is not empty -%}
       {%- if news_url is not empty -%}
-        <a {{ news_url_attributes }} href="{{ news_url }}" aria-hidden="true">
+        <a {{ news_url_attributes }} href="{{ news_url }}" aria-hidden="true" tabindex="-1">
```

Comment:
Didn't Kevin say this needed an `aria-hidden="true"` as well?

Reply:
I see it on line 71. But I am on the phone…

Reply:
On line 71, I see the link already had the `aria-hidden="true"`. I didn't have to add it.

Reply:
wow. i'm blind. Don't mind me.